operators/pkg/controller/elasticsearch/client/v6.go

```diff
@@ -38,8 +38,8 @@ func (c *clientV6) EnableShardAllocation(ctx context.Context) error {
 	return c.put(ctx, "/_cluster/settings", allocationSetting, nil)
 }
 
-func (c *clientV6) DisableShardAllocation(ctx context.Context) error {
-	allocationSetting := ClusterRoutingAllocation{AllocationSettings{Enable: "none"}}
+func (c *clientV6) DisableReplicasShardAllocation(ctx context.Context) error {
+	allocationSetting := ClusterRoutingAllocation{AllocationSettings{Enable: "primaries"}}
```

Comment:
It's "the official way" from the documentation since a few ES versions, and I think it makes sense (we don't have to refuse new indices if we plan to shut down a node).